### DISCUSSION TITLE
How to Configure SeleniumBase with an IPv6 Proxy?

### DISCUSSION THREAD

**Discussion by Muhammad-Awais-Ramzan** [seleniumbase/SeleniumBase#2771] [Q&A] [answered]
Hello everyone,

I'm currently working on a project where I need to configure SeleniumBase to use an IPv6 proxy for web automation tasks. I've found information on setting up proxies in SeleniumBase quite straightforward for IPv4, but I'm encountering some challenges with IPv6 proxies.

Could anyone share Python code examples or guidance on how to effectively set up Selenium to route traffic through an IPv6 proxy?

Thank you in advance for your help!

**Comment by mdmintz** [comment, marked as answer]
Hello, you can set proxies via the `proxy` arg in SeleniumBase. `host:port` or `user:pass@host:port` formats. If for some reason the current build is having trouble with IPv6, you can generate a Chrome extension with the proxy settings using this solution from StackOverflow: https://stackoverflow.com/a/53681091/7058266. Then, load the extension with SeleniumBase using `extension_zip` or `extension_dir` args.
